card/variables.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css2?family=Arvo&display=swap");
+
+html {
+  box-sizing: border-box;
+  font-family: "Arvo", serif;
+}
+
+body {
+  background-color: var(--bg-color);
+  margin: 0;
+  padding: 0;
+}
+
+body::-webkit-scrollbar {
+  width: 0.5rem;
+  background-color: var(--grey);
+}
+
+body::-webkit-scrollbar-thumb {
+  background-color: var(--primary-color);
+  border-radius: var(--radius);
+}
+
+:root {
+  --primary-color: #54bab9;
+  --secondary-color: #6b39ff;
+  --success-color: #42ff85;
+  --danger-color: #ef4b4b;
+  --warning-color: #f5b314;
+  --primary-bgcolor: #b5deff;
+  --secondary-bgcolor: #cab8ff;
+  --success-bgcolor: #c1ffd7;
+  --danger-bgcolor: #f9c0c0;
+  --warning-bgcolor: #fcffa6;
+  --white: #ffffff;
+  --black: #000000;
+  --grey: #6c7780;
+  --p-0: 0px;
+  --m-0: 0px;
+  --p-1: 0.25rem;
+  --m-1: 0.25rem;
+  --p-2: 0.5rem;
+  --m-2: 0.5rem;
+  --p-3: 0.75rem;
+  --m-3: 0.75rem;
+  --p-4: 1rem;
+  --m-4: 1rem;
+  --p-5: 1.25rem;
+  --m-5: 1.25rem;
+  --p-6: 1.5rem;
+  --m-6: 1.5rem;
+  --p-7: 1.75rem;
+  --m-7: 1.75rem;
+  --p-8: 2rem;
+  --m-8: 2rem;
+  --fontSize-0: 0.5rem;
+  --fontSize-1: 1rem;
+  --fontSize-2: 1.25rem;
+  --fontSize-3: 1.5rem;
+  --fontSize-4: 1.75rem;
+  --fontSize-5: 2rem;
+  --radius: 0.25rem;
+}
```

Comment:
Why not add these values common to all the components? That is, say make one variables.css for all the components?

Are these variables and resets only specific to this component?